### DISCUSSION TITLE
The new deployment not update on my domain, only update on automatic domain

### DISCUSSION THREAD

**Discussion by hungfhp** [vercel/community#2380] [Help] [answered]
### Summary

You can see the new code was deployed successfully.
But the new website only deployed to "Automatic URL". On "Custom Domain" still have old version.

You can check it by go to those sites:
https://bh-web-blog-bh-web.vercel.app
https://bh-web-blog-git-main-bh-web.vercel.app
https://bringhacks.com
https://www.bringhacks.com

![Screenshot 2023-05-10 at 2 00 50](https://github.com/vercel/community/assets/18446429/67273312-1e6e-4e55-9219-2fe7db871326)

DIFFERENT:
https://bh-web-blog-bh-web.vercel.app : NEW CODE
![Screenshot 2023-05-10 at 2 04 25](https://github.com/vercel/community/assets/18446429/3379827c-ba30-4c8a-b070-c20cb2f8d1c5)

https://bringhacks.com : STILL OLD CODE
![Screenshot 2023-05-10 at 2 04 33](https://github.com/vercel/community/assets/18446429/c6e919f7-2c5e-4040-ba22-f69dda8959bd)

**Comment by amyegan** [comment]
Hello @hungfhp! I'm not able to reproduce this problem in my test app. Do you have a minimal reproduction or public repo that I can use to locate the source of this problem?

**Comment by hungfhp** [comment]
Hi @amyegan,
I don't have public repo. Can you suggest some ideas about why it's happen?

**Comment by hungfhp** [comment, marked as answer]
Solved by remove the app & domain -> re-init
